### PR TITLE
Add websockets, ocpp module dependencies to test env

### PR DESCRIPTION
**Pull request by drc38**

### CODE REVIEW

**Comment by drc38** [comment]
Not sure how to resolve this error `homeassistant.loader.IntegrationNotFound: Integration 'ocpp' not found.`

**Comment by drc38** [comment]
@lbbrhzn , I think I've gone as far as I can. Still a few more issues to resolve with the testing scripts

**Comment by lbbrhzn** [comment]
thnx drc38! Can we remove the test that is still failing?

File: .github/workflows/constraints.txt [hunk 1]
```diff
@@ -1,5 +1,8 @@
-pip==21.1.3
+pip<=20.3
```
Comment (lbbrhzn) [inline .github/workflows/constraints.txt:1]: Is this <=20.3 coming from home assistant?

Comment (drc38) [reply .github/workflows/constraints.txt:1]: Yes it was

**Comment by drc38** [comment]
@lbbrhzn , tests are now running without error but failing as there is not sufficient code coverage

**Comment by lbbrhzn** [comment]
Ok, good enough for now!

**Pull request merged**